packages/shopify-codemod/transforms/assert/assert-to-assert-ok.js

```diff
@@ -0,0 +1,27 @@
+export default function assertToAssertOk({source}, {jscodeshift: j}, {printOptions = {}}) {
+  return j(source)
+    .find(j.CallExpression, {
+      callee: {
+        type: j.Identifier.name,
+        name: 'assert',
+      },
+      arguments: {
+        0: {
+          type: (type) => [
+            j.BinaryExpression.name,
+            j.CallExpression.name,
+            j.Identifier.name,
+            j.MemberExpression.name,
+            j.UnaryExpression.name,
+          ].indexOf(type) > -1,
```

Comment:
Might be nicer to use a `Set` in scope instead of an array constructed each time.